libs/blocks/brick/brick.css

```diff
@@ -67,7 +67,7 @@
   align-items: flex-start;
 }
 
-.brick .background div,
+.brick .background div:not([class^="picture-meta"]),
```

Comment:
Since this `picture-meta` element would never be shown to end consumers, I'd suggest we counteract it in the `preflight` styles. Same applies for the declaration on L105.